frontend/src/component/changeRequest/changeRequest.types.ts

```diff
@@ -150,7 +161,9 @@ export type IFeatureChange =
     | IChangeRequestEnabled
     | IChangeRequestPatchVariant
     | IChangeRequestReorderStrategy
-    | IChangeRequestArchiveFeature;
+    | IChangeRequestArchiveFeature
+    | IChangeRequestAddDependency
```

Comment:
Ideally I'd like those to come from openapi. I'll try to use openapi/orval types in another PR.